app/apollo/resolvers.ts

```diff
@@ -1,7 +1,9 @@
 // Write resolvers to respond to our queries and mutations
+// https://www.notion.so/neo4j-graphql-v1-0-0-alpha-2-d47908030d4e4a0c86babbaef63887d0
```

Comment:
This was an excellent link provided during the 2021.01.14 Neo4j GraphQL Community Call